src/rpc/RPCHelpers.cpp

```diff
@@ -372,7 +372,11 @@ toJson(ripple::LedgerHeader const& lgrInfo, bool const binary)
         header[JS(close_time_resolution)] = lgrInfo.closeTimeResolution.count();
         header[JS(close_time_iso)] = ripple::to_string_iso(lgrInfo.closeTime);
         header[JS(ledger_hash)] = ripple::strHex(lgrInfo.hash);
-        header[JS(ledger_index)] = std::to_string(lgrInfo.seq);
+        if (apiVersion < 2) {
```

Comment:
Let's use `2u` so we don't get a warning

Reply:
I'd also maybe move this if/else block to the bottom and have a space between the block and the rest of the code.